server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/Serializer.scala

```diff
@@ -53,4 +53,8 @@ class Serializer {
     (m: Map[Id, Any]) => {
       JsObject(m.map { case (k, v) => (k.value, vr.writes(v)) }.toSeq)
     }
+
+  def serialize(session: Session): String = {
+    sessionWrites.writes(session).toString()
```

Comment:
you should not really on `toString` unless there are no other solution. In this case you should use `Json` methods